src/mongocxx/test/versioned_api.cpp

```diff
@@ -26,8 +27,23 @@ using namespace mongocxx;
 // We'll format many of these examples by hand
 // clang-format off
 
+static bool is_server_v5_or_newer() {
+    using namespace bsoncxx::builder::basic;
+    client cl{uri{}};
+    auto status = cl["admin"].run_command(make_document(kvp("serverStatus", 1)));
```

Comment:
This may have the same caveat as running `run_command` for the `count` command below. The client above can be configured with a conditional version API like so:

```
client mongodb_client{uri{}, test_util::add_test_server_api()};
```

But it may be better to use the `test_util::get_max_wire_version` helper altogether.